_posts/2025-02-14-this-month-in-servo.md

```diff
@@ -248,59 +303,78 @@ https://github.com/servo/servo/pull/35098	(@atouchet, #35098)	Create servo-media
+    api
 -https://github.com/servo/servo/pull/35150	(@dependabot[bot], @dependabot[bot], #35150)	build(deps): bump unicode-ident from 1.0.14 to 1.0.15 (#35150)
 -https://github.com/servo/servo/pull/35149	(@dependabot[bot], @dependabot[bot], #35149)	build(deps): bump objc2-encode from 4.0.3 to 4.1.0 (#35149)
-https://github.com/servo/servo/pull/35066	(@shanehandley, #35066)	Update the FormData constructor to allow providing a submitter (#35066)
-https://github.com/servo/servo/pull/35147	(@mrobinson, #35147)	Fix some clippy issues on the Android build (#35147)
-https://github.com/servo/servo/pull/35129	(@Loirooriol, #35129)	layout: Hide collapsed borders crossed by spanning cells (#35129)
-https://github.com/servo/servo/pull/35125	(@Loirooriol, #35125)	layout: simplify `CollapsedBorderLine` (#35125)
-https://github.com/servo/servo/pull/35118	(@delan, @mrobinson, #35118)	Add minimal libservo example using winit (#35118)
-https://github.com/servo/servo/pull/35122	(@Loirooriol, #35122)	layout: Fix table geometry when collapsed borders have different sizes (#35122)
-https://github.com/servo/servo/pull/35116	(@delan, @mrobinson, #35116)	Fix building libservo with `cargo build -p libservo` (#35116)
++https://github.com/servo/servo/pull/35066	(@shanehandley, #35066)	Update the FormData constructor to allow providing a submitter (#35066)
+    api
+-https://github.com/servo/servo/pull/35147	(@mrobinson, #35147)	Fix some clippy issues on the Android build (#35147)
++https://github.com/servo/servo/pull/35129	(@Loirooriol, #35129)	layout: Hide collapsed borders crossed by spanning cells (#35129)
+    layout; table ‘border-collapse’ painting
+-https://github.com/servo/servo/pull/35125	(@Loirooriol, #35125)	layout: simplify `CollapsedBorderLine` (#35125)
++https://github.com/servo/servo/pull/35118	(@delan, @mrobinson, #35118)	Add minimal libservo example using winit (#35118)
+    webview
++https://github.com/servo/servo/pull/35122	(@Loirooriol, #35122)	layout: Fix table geometry when collapsed borders have different sizes (#35122)
+    layout; table ‘border-collapse’ layout
++https://github.com/servo/servo/pull/35116	(@delan, @mrobinson, #35116)	Fix building libservo with `cargo build -p libservo` (#35116)
+    webview
 -https://github.com/servo/servo/pull/35143	(@jschwe, #35143)	ci: Fix ohos test artifact name (#35143)
 >>> 2025-01-25T06:12:33Z
 +https://github.com/servo/servo/pull/35148	(@webbeef, #35148)	Add support for HTMLStyleElement.media (#35148)
     api
-https://github.com/servo/servo/pull/35168	(@willypuzzle, #35168)	[35131] Borrow hazard in Document::broadcast_active_resize_observations (#35168)
-https://github.com/servo/servo/pull/35145	(@Taym95, #35145)	Use Dom in native promise handlers instead of Trusted (#35145)
+-https://github.com/servo/servo/pull/35168	(@willypuzzle, #35168)	[35131] Borrow hazard in Document::broadcast_active_resize_observations (#35168)
```

Comment:
This commit fixed a crash.

Reply:
fixed in c5cc5ed58bdb1cf750cedd59769a5d5a0dcc7b64